src/FailureReceiver/FailureReceiverName.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace KaroIO\MessengerMonitorBundle\FailureReceiver;
+
+/**
+ * @internal
+ */
+class FailureReceiverName
+{
+    private $failureReceiverName;
+
+    public function __construct(?string $failureReceiverName = null)
```

Comment:
I don't think the arg needs to be optional. Even if there is no failure receiver, we still pass null to this I believe in the compiler pass.